WordPress/WordPressUITests/Flows/LoginFlow.swift

```diff
@@ -5,8 +5,6 @@ class LoginFlow {
 
     @discardableResult
     static func login(email: String, password: String) throws -> MySiteScreen {
-        try logoutIfNeeded()
```

Comment:
if we're no longer using this, what do you think about removing the function and all its references from the UI tests? i.e. on [LoginFlow](https://github.com/wordpress-mobile/WordPress-iOS/blob/b006c4a987537536af39452bd175c4654992cfd5/WordPress/WordPressUITests/Flows/LoginFlow.swift#L67) and on [this comment](https://github.com/wordpress-mobile/WordPress-iOS/blob/b006c4a987537536af39452bd175c4654992cfd5/WordPress/WordPressUITests/Tests/LoginTests.swift#L93)